src/Instances/ListL/ConcatenationMonoid.php

```diff
@@ -29,6 +29,6 @@ public function mempty(): array
      */
     public function append($a, $b)
     {
-        return array_merge($b, $a);
+        return array_merge($a, $b);
```

Comment:
Is this intentional ?

Reply:
yes, it is. I'll write an ADR for this.

Keeping the arguments in the same order is just more natural.

Otherwise `append(['a'], ['b'])` would be `['b', 'a']`, while it is more expected to be `['a', 'b']`.

Do you agree?

Reply:
Yes, definitely ! Thanks for the explanation :)